cross/syncstorage-rs/Makefile

```diff
@@ -24,6 +24,8 @@ CARGO_BUILD_ARGS += --locked
 DEPENDS = cross/openssl3
 # for mysqlclient
 DEPENDS += cross/mysql-connector-c
+ENV += MYSQLCLIENT_CFLAGS="$(CFLAGS) -I$(STAGING_INSTALL_PREFIX)/include/mysql -I$(STAGING_INSTALL_PREFIX)/$(PYTHON_INC_DIR)"
+ENV += MYSQLCLIENT_LDFLAGS="$(LDFLAGS)"
```

Comment:
I guess this is one of those anomalies that breaks the self-contained dependencies rule, since this line is in the cross makefile:

```
DEPENDS += cross/mysql-connector-c
```